kamailio/users/config/kamailio.cfg

```diff
@@ -195,6 +196,11 @@ loadmodule    "pike.so"
 loadmodule    "ndb_redis.so"
 #!endif
 
+# DMQ
+modparam("dmq", "server_address", "sip:users.ivozprovider.local:5060")
+modparam("dmq", "notification_address", "sip:trunks.ivozprovider.local:5060")
+modparam("dmq", "ping_interval", 3600)
```

Comment:
I know you couldn't care less but this is not compatible with single public IP standalone, because trunks UDP listen address is 7060.

Reply:
Good catch!